src/site_assessment/best_fit_polygons.jl

```diff
@@ -35,6 +35,13 @@ positive.
 - Highest scoring rotation step
 - Highest scoring polygon
 - Quality control flag for site, indicating if `surr_threshold` was met in the highest scoring rotation.
+
+# Extended help
+The scores produced are a proportion of the polygon that are covered by valid pixel points,
+relative to the maximum number of points (`max_count`) (0-1). `max_count` is approximate
+(determined by user `x_dist` and `y_dist` box dimensions) and doesn't account for buffering
+and rotation of the search box. In rare cases scores could be > 1, however returned values
+are capped at max 1.
```

Comment:
As mentioned elsewhere, Extended help should directly below the main description text.

`# Returns` is typically the last section in a docstring

Reply:
As an aside, didn't we update the description for `rel_pix` in the argument list to at least indicate what `rel` means?

Reply:
Addressed second comment in #39